packages/flutter_tools/lib/src/isolated/devtools_launcher.dart

```diff
@@ -2,11 +2,15 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-import 'package:devtools_server/devtools_server.dart' as devtools_server;
```

Comment:
Since the import of devtools_server is being removed, this file may no longer need to live under `isolated/`. We'd have to check that the import of `browser_launcher` doesn't mean that it has to stay though.

Reply:
Moved devtools_launcher out of isolated. `browser_launcher` was already in the pubspec before my PR. Is this enough indication that it is fine living outside of isolated?

Reply:
Thanks. Should be. I'll check.